metabase/metabase
# Convert NewsletterForm to TypeScript

Epic: https://github.com/metabase/metabase/issues/19247

Converts `NewsletterForm` to typescript and styled-components.

How to test:
- Run a fresh instance
- The last step in the setup flow asks the user to subscribe to the newsletter
- Make sure the request succeeds
